spec/services/listing_service_spec.rb

```diff
@@ -57,4 +57,167 @@
       end
     end
   end
+
+  it 'gets listings given user id', :vcr do
+    # json_response = File.read('spec/fixtures/user_listings_given_user_id.json')
+    json_body = "{\"data\":[{\"id\":\"3900\",\"type\":\"listing\",\"attributes\":{\"year\":2019,\"make\":\"Audi\",\"model\":\"A4\",\"trim\":\"XL\",\"body\":\"Cargo Van\",\"transmission\":\"CVT\",\"vin\":\"3RLGLK247RIU45602\",\"state\":\"DE\",\"condition\":1.14,\"odometer\":69615,\"color\":\"Beige\",\"interior\":\"Sit aspernatur aut porro.\",\"sellingprice\":11719}},{\"id\":\"3901\",\"type\":\"listing\",\"attributes\":{\"year\":2013,\"make\":\"Chevy\",\"model\":\"Silverado\",\"trim\":\"ESi\",\"body\":\"Sedan\",\"transmission\":\"Automanual\",\"vin\":\"35AOE1VBJI0768454\",\"state\":\"ND\",\"condition\":3.74,\"odometer\":207221,\"color\":\"Orange\",\"interior\":\"Perferendis saepe ut quos.\",\"sellingprice\":36857}},{\"id\":\"3902\",\"type\":\"listing\",\"attributes\":{\"year\":2017,\"make\":\"Dodge\",\"model\":\"Challenger\",\"trim\":\"XLE\",\"body\":\"Passenger Van\",\"transmission\":\"Automatic\",\"vin\":\"21VN2P5P46EW95516\",\"state\":\"TN\",\"condition\":4.07,\"odometer\":248733,\"color\":\"Beige\",\"interior\":\"Ut minima delectus qui.\",\"sellingprice\":23207}},{\"id\":\"3903\",\"type\":\"listing\",\"attributes\":{\"year\":2019,\"make\":\"Ford\",\"model\":\"Mustang\",\"trim\":\"L\",\"body\":\"Minivan\",\"transmission\":\"Manual\",\"vin\":\"2EKO3C8Q2OXU03372\",\"state\":\"AL\",\"condition\":2.27,\"odometer\":132240,\"color\":\"Grey\",\"interior\":\"Eaque ut sed nemo.\",\"sellingprice\":34891}}]}"
+
+    stub_request(:get, "http://localhost:3000/api/v1/users/1/listings").to_return(status: 200, body: json_body, headers: {})
+
+    user_listings = ListingService.listings_by_user_id(1)
+    expect(user_listings).to be_a(Hash)
+    expect(user_listings[:data]).to be_a(Array)
+    expect(user_listings[:data][0][:attributes][:year]).to be_an(Integer)
+    expect(user_listings[:data][0][:attributes][:make]).to be_an(String)
+    expect(user_listings[:data][0][:attributes][:model]).to be_an(String)
+    expect(user_listings[:data][0][:attributes][:make]).to eq('Audi')
+    expect(user_listings[:data][0][:attributes][:model]).to eq('A4')
+  end
 end
+  ##Supposed to get listing based on filter params(Big One) && one based on listing id
```

Comment:
we should probably delete this if it is no longer needed.

Reply:
Yea we should do that later once we figure out heroku and get OAuth. For now this will suffice for functionality I believe